### PR TITLE
Create index page branch

### PR DESCRIPTION
**Pull request by cassking**
Finished enzyme tests for PlayerTile, PlayerShow, PlayerContainer

### CODE REVIEW

File: app/javascript/components/PlayerTile.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Link } from 'react-router';
 
 const PlayerTile = props => {
+  console.log('hello tile')
```
Comment (jdmacmurtrie) [inline app/javascript/components/PlayerTile.js]: Remove this line.

File: spec/javascript/components/PlayerShowTest.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (jdmacmurtrie) [inline spec/javascript/components/PlayerShowTest.js]: Remove this blank line.

File: spec/javascript/containers/PlayerContainerTest.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import PlayerTile from '../../../app/javascript/components/PlayerTile';
+
+describe('PlayerContainer', () => {
+  let wrapper;
+  let player1 = {
+    player: [{
+      first_name: 'James',
+      last_name: 'Harden',
+      avatar_url: 'https://specials-images.forbesimg.com/imageserve/5936925ea7ea434078d4c5eb/416x416.jpg?background=000000&cropX1=1335&cropX2=3965&cropY1=104&cropY2=2735',
+
```
Comment (jdmacmurtrie) [inline spec/javascript/containers/PlayerContainerTest.js]: Remove this line, and the comma at the end of line 11

File: spec/javascript/containers/PlayerContainerTest.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      let response = new Response(responseBody, {
+        status: '200',
+        statusText: 'OK',
+        headers: { 'Content-Type': 'application/json' }
+      });
+      return Promise.resolve(response);
+    });
+
+    wrapper = mount(<PlayerContainer />);
+
```
Comment (jdmacmurtrie) [inline spec/javascript/containers/PlayerContainerTest.js]: Remove this empty line.

Comment (cassking) [reply spec/javascript/containers/PlayerContainerTest.js]: fixed

File: spec/javascript/containers/PlayerContainerTest.js (hunk 3 above)
Comment (cassking) [reply spec/javascript/containers/PlayerContainerTest.js]: fixed

File: spec/javascript/components/PlayerShowTest.js (hunk 2 above)
Comment (cassking) [reply spec/javascript/components/PlayerShowTest.js]: fixed

File: app/javascript/components/PlayerTile.js (hunk 1 above)
Comment (cassking) [reply app/javascript/components/PlayerTile.js]: fixed

Comment (cassking) [reply app/javascript/components/PlayerTile.js]: removed

File: spec/javascript/containers/PlayerContainerTest.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        statusText: 'OK',
+        headers: { 'Content-Type': 'application/json' }
+      });
+      return Promise.resolve(response);
+    });
+  
+    wrapper = mount(<PlayerContainer />);
+  });
+
+  fit('should have the specified intial state', () => {
```
Comment (jdmacmurtrie) [inline spec/javascript/containers/PlayerContainerTest.js]: unfocus this!

File: spec/javascript/containers/PlayerContainerTest.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  });
+
+  it('should render a h4 tag', (done) => {
+     setTimeout(() => {
+       expect(wrapper.find('h4')).toBePresent();
+       done();
+     }, 0)
+   });
+
+   fit('should render the PlayerTile with different props, when players is not an empty array', (done) => {
```
Comment (jdmacmurtrie) [inline spec/javascript/containers/PlayerContainerTest.js]: and here!

File: spec/javascript/containers/PlayerContainerTest.js (hunk 5 above)
Comment (cassking) [reply spec/javascript/containers/PlayerContainerTest.js]: unfocused and tsts pass

File: spec/javascript/components/PlayerShowTest.js (hunk 2 above)
Comment (cassking) [reply spec/javascript/components/PlayerShowTest.js]: killed it

**Pull request merged**